src/components/rmrk/Gallery/Item/Sharing.vue

```diff
@@ -1,7 +1,12 @@
 <template>
   <div class="share">
     <b-field position="is-right">
+      <div>
+        <slot />
```

Comment:
if you put just slot here (omit `<div>` ) does it break the layout?

Reply:
`<div>` is used here to remove spacing between button